src/main/ruby/truffleruby/core/process.rb

```diff
@@ -729,19 +731,20 @@ class Status
     attr_reader :termsig
     attr_reader :stopsig
 
-    def initialize(pid=nil, status=nil, termsig=nil, stopsig=nil)
+    def initialize(pid=nil, status=nil, termsig=nil, stopsig=nil, raw_status=nil)
       @pid = pid
       @status = status
```

Comment:
I think we should rename this to `@exitstatus` to avoid confusion and mirror the C macro names.